### PR TITLE
Build request: pass yum_proxy value to workers

### PR DESCRIPTION
**Pull request by vrutkovs**
Signed-off-by: Vadim Rutkovsky <vrutkovs@redhat.com>

### CODE REVIEW

File: osbs/build/build_request.py [hunk 1]
```diff
@@ -249,6 +249,7 @@ def render_orchestrate_build(self):
             'sources_command': self.spec.sources_command.value,
             'vendor': self.spec.vendor.value,
             'equal_labels': equal_labels_string,
+            'yum_proxy': self.spec.yum_proxy.value
```
Comment (twaugh) [inline osbs/build/build_request.py]: Nicer if this ends in a comma.

File: tests/build_/test_build_request.py [hunk 2]
```diff
@@ -1164,6 +1164,7 @@ def test_render_optional_plugins(self, tmpdir):
             'sources_command': 'sources_command',
             'vendor': 'vendor',
             'equal_labels': [['label1', 'label2'], ['label3', 'label4']],
+            'yum_proxy': 'http://proxy:3128'
```
Comment (twaugh) [inline tests/build_/test_build_request.py]: Nicer if this ends in a comma.

Comment (vrutkovs) [inline tests/build_/test_build_request.py]: Right, fixed both

**Comment by lcarva** [comment]
Looks good, but there are conflicts due to latest merges.

**Comment by vrutkovs** [comment]
Rebased

**Comment by lcarva** [comment]
Updated release notes.

**Pull request merged**